toolkit/tools/imagegen/attendedinstaller/views/diskview/manualpartitionwidget/manualpartitionwidget.go

```diff
@@ -127,26 +130,13 @@ func (mp *ManualPartitionWidget) Initialize(backButtonText string, sysConfig *co
 
 	mp.addPartitionForm = tview.NewForm().
 		SetButtonsAlign(tview.AlignCenter)
+	mp.formatInput, mp.formatInputReset = mp.enumInputBox(validPartitionFormats)
+	mp.formatInput = mp.formatInput.SetLabel(uitext.DiskFormatLabel).
+		SetFieldBackgroundColor(tcell.ColorWhite)
 
-	// Dropdowns do not expose the functions needed to override the list colors.
-	// Alter the defaults now so they are captured by the dropdowns and then restore the style
-	// for future elements.
-	originalStyle := tview.Styles
-	tview.Styles.ContrastBackgroundColor = tcell.ColorWhite
-	tview.Styles.MoreContrastBackgroundColor = tcell.ColorBlack
-	tview.Styles.PrimitiveBackgroundColor = tcell.ColorWhite
-	tview.Styles.PrimaryTextColor = tcell.ColorBlue
-
-	mp.formatDropDown = tview.NewDropDown().
-		SetLabel(uitext.DiskFormatLabel).
-		SetOptions(validPartitionFormats, nil)
-
-	mp.sizeUnitDropDown = tview.NewDropDown().
-		SetLabel(uitext.DiskSizeUnitLabel).
-		SetOptions(validSizeUnits, nil)
-
-	// Restore the global style
-	tview.Styles = originalStyle
+	mp.sizeUnitInput, mp.sizeUnitInputReset = mp.enumInputBox(validSizeUnits)
+	mp.sizeUnitInput = mp.sizeUnitInput.SetLabel(uitext.DiskSizeUnitLabel).
```

Comment:
Nit: could we follow the format from the deleted lines where each `Set*()` function was called on a separate line?